### PR TITLE
signup page layout and squeezed fixed

### LINKED ISSUE #469: Improve style of Signup form
**Actual Behaviour**

The borders of the Signup form fields aren't styled, and not enough spacing between the form fields and the text.

**Expected Behaviour**

Style to borders to be consistent with the Login Form. Add some spacing between the form fields and the text

**Screenshots of the issue**

screenshot 18

**Would you like to work on the issue?**

Yes I would like to work on the issue

### PR DESCRIPTION
**Pull request by pg07codes**
Fixes #469 
image

fixed the layout of the signup page. And also the styling as well.
Deployed at [surge here](https://skillful-war.surge.sh)
This PR is new one of the earlier already reviewed PR #359 . @DNS-404 recommended me to make a new PR.
 @PrP-11 , @anshumanv , @AnupKumarPanwar please Review

### CODE REVIEW

**Comment by rossoskull** [comment]
Issue #469 is about the border style of the form fields of the signup form, and the spacing between the form fields and the text. In your PR, you have fixed the layout.

**Comment by pg07codes** [comment]
couldn't resolve this issue . @DNS-404 help

File: src/components/Auth/SignUp/SignUp.react.js [hunk 1]
```diff
@@ -286,10 +286,13 @@ export default class SignUp extends Component {
                   labelColor="#fff"
                 />
               </div>
-              <br />
+          <br />
```
Comment (fragm3) [inline src/components/Auth/SignUp/SignUp.react.js]: Insert `····`

File: src/components/Auth/SignUp/SignUp.react.js [hunk 2]
```diff
@@ -286,10 +286,13 @@ export default class SignUp extends Component {
...
               <h1 style={{ textAlign: 'center' }}>OR</h1>
+
```
Comment (fragm3) [inline src/components/Auth/SignUp/SignUp.react.js]: Remove extra new line

**Comment by pg07codes** [comment]
thanks to @DNS-404 and @fragm3 . Fixed Travis.

**Pull request merged**